README.md

```diff
@@ -124,6 +124,16 @@ BibTeX entry:
 }
 ```
 
+If you use AutoGluon's time series forecasting functionality in a scientific publication, please cite the following paper:
+```bibtex
+@inproceedings{agtimeseries,
+  title={{AutoGluon-TimeSeries}: {AutoML} for Probabilistic Time Series Forecasting},
+  author={Shchur, Oleksandr and Turkmen, Caner and Erickson, Nick and Shen, Huibin and Shirkov, Alexander and Hu, Tony and Wang, Yuyang},
+  booktitle={International Conference on Automated Machine Learning},
+  year={2023}
+}
+```
+
```

Comment:
Should we consider shortening our `README.md` and, for example, moving the bibtex references to a dedicated page on auto.gluon.ai? 

I think that now the readme feels a bit crowded.

Reply:
Good point, I've added this as a TODO to myself.